src/voice-pe/customizing/automations-scripts.md

```diff
@@ -12,7 +12,10 @@ You can use scripts and automations to control the device. The steps below show
 <iframe width="560" height="315" src="https://www.youtube.com/embed/QhhuCzbQEag" videotitle="Use automations and scripts to control Home Assistant Voice Preview Edition" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" controls></iframe>
 </div>
 
-## Example 1: Playing announcements on the device
+{% steps %}
+{% step "Playing announcements on the device" "Example 1:" %}
```

Comment:
This seems weird to me, the prefix is always followed by `:` right?
If so, that should be in the `step` code, not something that needs to be added to each use.

Reply:
no it's not. with "normal" steps, there is no separator.

Reply:
fixed in https://github.com/NabuCasa/support/pull/521